dnf5/commands/repoquery/repoquery.cpp

```diff
@@ -271,7 +271,7 @@ void RepoqueryCommand::set_argument_parser() {
     recent = std::make_unique<libdnf5::cli::session::BoolOption>(
         *this, "recent", '\0', "Limit to only recently changed packages.", false);
     installonly = std::make_unique<libdnf5::cli::session::BoolOption>(
-        *this, "installonly", '\0', "Limit to installed installonly packages.", false);
```

Comment:
I think that's not a typo, but on purpose to emphasize that only `installed` packages are considered.

Reply:
I see, you are correct. I will drop the commit.
Do you have any idea why is that? The filter would make sense to me also on available packages. Plus it can be combined with `--installed`.

Reply:
I think the filter was implemented according to dnf4 version which behaves the same.
But you are right I guess.